search-app/src/cmr/search/services/query_execution/facets/facets_v2_results_feature.clj

```diff
@@ -152,12 +152,14 @@
         concept-type (:concept-type query)
         facet-fields (:facet-fields query)
         facet-fields (if facet-fields facet-fields (facets-v2-params concept-type))
-        query-params (parse-params query-string "UTF-8")]
+        query-params (parse-params query-string "UTF-8")
+        facets-size (get query-params "facets_size")
```

Comment:
We should support both `facets_size` and `facets-size` params. We are already parsing out facet-fields in the query. We should perhaps put the facets_size into the query as well and save us the trouble to have to parse out the facets_size param again. It will make a lot more sense especially we need to support individual facets size for each facet field.

Reply:
1. I'm not sure why we are not supporting both include_facets and include-facets, but we want to support both facets_size and facets-size?  
2. facet-fields are just the (:science-keywords :platform :instrument :data-center :project :processing-level-id :variables), I could add facets_size to it but it still doesn't have values. Am I misunderstanding you?  (did you mean that facet-fields already contain all the facet parameter values?) I'm not sure what you mean.